packages/esm-patient-chart-app/src/patient-banner-tags/visit-attribute-tags.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Tag } from '@carbon/react';
+import { formatDate, useConfig } from '@openmrs/esm-framework';
+import { ChartConfig } from '../config-schema';
+import { useVisitOrOfflineVisit } from '@openmrs/esm-patient-common-lib';
+
+interface VisitAttributeTagsProps {
+  patientUuid: string;
+}
+
+const getAttributeValue = (attributeType, value) => {
+  switch (attributeType?.datatypeClassname) {
+    case 'org.openmrs.customdatatype.datatype.ConceptDatatype':
+      return value?.display;
+    case 'org.openmrs.customdatatype.datatype.FloatDatatype':
+      return value;
+    case 'org.openmrs.customdatatype.datatype.FreeTextDatatype':
+      return value;
+    case 'org.openmrs.customdatatype.datatype.LongFreeTextDatatype':
+      return value;
+    case 'org.openmrs.customdatatype.datatype.BooleanDatatype':
+      return value;
```

Comment:
It looks as though all these `case` clauses return the same value as the `default` clause. Is this intentional? Should there maybe be an empty state in the `default` clause (something like `--`)? If so, that bit of code could be shortened to:

```suggestion
    case 'org.openmrs.customdatatype.datatype.FloatDatatype':
    case 'org.openmrs.customdatatype.datatype.FreeTextDatatype':
    case 'org.openmrs.customdatatype.datatype.LongFreeTextDatatype':
    case 'org.openmrs.customdatatype.datatype.BooleanDatatype':
      return value;
```